app/src/main/java/org/traccar/client/StatusWidget.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016 - 2022 Anton Tananaev (anton@traccar.org)
```

Comment:
Update the date. Also feel free to put your name / email if you want.

Reply:
`Copyright 2016 - 2023 Anton Tananaev (anton@traccar.org), Anton-V-K` - OK?